src/routes/lend.js

```diff
@@ -48,8 +47,9 @@ function LendPage() {
     }, [setNFTsInUserWallet, setError, listNFT]);
 
     const loadFakeNFT = useCallback(async () => {
+        const abiManager = new ABIManager(LoginService.getInstance().chainId);
         const signer = LoginService.getInstance().signer;
-        const contract = new ethers.Contract(ContractAddress.FakeNFT, FakeNFTContract.abi, signer);
+        const contract = new ethers.Contract(abiManager.ContractAddress().FakeNFT, abiManager.FakeNFTContract().abi, signer);
```

Comment:
Looking great! the only minor thing I would add is to put this call inside abiManager and just do something like ```const contract = abiManager.getFakeNFTContract(signer)```